src/components/UIUC-Components/EmailChipsComponent.tsx

```diff
@@ -42,7 +43,7 @@ const EmailChipsComponent = ({
         metadata.is_private = JSON.parse(
           metadata.is_private as unknown as string,
         )
-        setEmailAddresses(metadata.approved_emails_list)
+        setEmailAddresses(metadata.approved_emails_list || [])
```

Comment:
hopefully this is fine... always nervous when we introduce new defaults. 

Instead of (`my_value || default`) I prefer (`my_value as MyExpectedType`). Both solve the type issue, but the `as` keywords ensure type issues are solved at compile time, instead of hiding behind defaults.